src/i18n/en-US/modelPlan/modelToOperations.ts

```diff
@@ -484,7 +484,7 @@ export const modelToOperationsMisc: Record<string, any> = {
         '{{count}} milestones associated with this solution',
       editMilestones: 'Edit related milestones',
       noMilestones:
-        'You haven’t selected any milestones or IT systems to implement this milestone.',
+        'You haven’t identified pinkSolutionForm/index.tsany specific milestones that this solution will help implement.',
```

Comment:
I think you copied and pasted the wrong thing there

Reply:
Ooop lol moving too fast!